flow/Makefile

```diff
@@ -460,6 +460,12 @@ UNSET_AND_MAKE = @bash -c 'for var in $(ISSUE_VARIABLES_NAMES); do unset $$var;
 #
 # A "do-synth" step would be welcomed, but it isn't strictly necessary
 # for the Bazel use-case.
+#
+# do-floorplan, do-place, do-cts, do-route, do-finish are the
+# supported interface to execute those stages without checking
+# for dependencies.
+#
+# The do- substeps of each of these stages are subject to change.
 define do-step
```

Comment:
It would be nice to have some comments about the arguments.  I can back infer it from the actual rule but it is painful.  Same with do-copy.